src/pages/LandingPage.tsx

```diff
@@ -52,61 +52,61 @@ export const LandingPage: React.FC<Props> = () => {
                 </Row>
             </Space>
                 <Row gutter={[24,24]} className={styles.TeamProfileContainer} justify='center'>
-                    <Col className={styles.TeamProfileRow}>
+                <Col className={styles.TeamProfileRow}>
                         <MemberProfile
-                            name="I am a name"
-                            position="This is a position"
-                            photo={placeholder}
-                            aria-label="Description of photo"
+                            name="Elif Coskun"
+                            position="Public Health Master's Student"
+                            photo='/Photos/Elif.jpg'
```

Comment:
Importing photos with a relative path is dangerous as if we need to change the source location it means changing it everywhere in the code. Plus, CRA does magic for us and photos can go in `/src` to help with some optimizations. See [here](https://create-react-app.dev/docs/adding-images-fonts-and-files/) for details.

Instead, import and manage in a similar way to how the paceholder was done. I think you may be able to remove the need for doing the explicit creation of image components as it current in `/assets/assets.index.ts` see [here](https://stackoverflow.com/questions/52759220/importing-images-in-typescript-react-cannot-find-module) for details.

Reply:
Gotcha, I'll consult best practices going forward and get started on fixing this today

Reply:
I ended up not having as much time as I thought, so I just created the image components manually. But I can easily come back to it in case this is something that should be changed in the future